src/Livewire/FolderTree.php

```diff
@@ -50,10 +52,31 @@ public function loadModel(array $arguments): void
     public function getRules(): array
     {
         return [
-            'files.*' => 'required|file|max:10240',
+            'files.*' => 'required|file|max:1024',
```

Comment:
get the max upload file size from livewire config config('livewire.temporary_file_upload.rules')